ersilia/hub/fetch/fetch.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         config_json=None,
         credentials_json=None,
         overwrite=None,
-        repo_path=None,
+        from_dir=None,
```

Comment:
I would recommend using the value of `from_dir` from the CLI to set both `from_dir`, and `repo_path` parameters for the `ModelFetcher` constructor.